Modules/Loadable/Sequences/Widgets/qMRMLSequenceEditWidget.cxx

```diff
@@ -65,6 +65,7 @@ class qMRMLSequenceEditWidgetPrivate
   vtkWeakPointer<vtkMRMLSequenceNode> SequenceNode;
   // data node class name that was used for populating the candidate node list
   QString DataNodeCandidatesClassName;
+  bool DataNodeCandidatesNeedsUpdate{true};
```

Comment:
Should the variable be called `DataNodeCandidatesNeedUpdate` ? (since there are more than one candidates)

Reply:
Thanks, fixed.